db/kvstore_versioned.go

```diff
@@ -47,4 +50,86 @@ type (
 		// SetVersion sets the version, and returns a KVStore to call Put()/Get()
 		SetVersion(uint64) KVStore
 	}
+
+	// KvWithVersion wraps the versioned DB implementation with a certain version
+	KvWithVersion struct {
+		db      VersionedDB
+		version uint64      // the current version
+		vns     []Namespace // versioned namespace
```

Comment:
`vns` is not used

Reply:
it is used in `func NewKVStoreWithVersion()` at L74 to pass the versioned namespace to creating the new versioned bolt db